lib/model/folder_sendrecv.go

```diff
@@ -1092,6 +1092,20 @@ func (f *sendReceiveFolder) renameFile(cur, source, target protocol.FileInfo, sn
 // handleFile queues the copies and pulls as necessary for a single new or
 // changed file.
 func (f *sendReceiveFolder) handleFile(file protocol.FileInfo, snap *db.Snapshot, copyChan chan<- copyBlocksState) {
+	
+	available := false
+	for _, dev := range snap.Availability(file.Name) {
+		if !f.model.IsFolderPausedOnDevice(f.folderID, dev) {
+		available = true
```

Comment:
Where `handleFile` is called there's already a check for availability - would make sense to add the pause check there.